notebooks/introductory_infilling.ipynb

```diff
@@ -0,0 +1,2572 @@
+    "# # an alternative, pop-up interface is shown if you uncomment the line below\n",
+    "#silicone.database_crunchers.base._DatabaseCruncher.derive_relationship?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These crunchers are best explored by looking at the examples below.\n",
+    "\n",
+    "### Closest Root Mean Square cruncher\n",
+    "\n",
+    "This cruncher finds the scenario and model in the infiller database that minimises the RMS distance between the the lead emissions in the infiller and infillee data. The follow data is then directly taken from that scenario. \n",
+    "The full documentation is as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "    Database cruncher which uses the 'closest RMS' technkque.\n",
+      "\n",
+      "    This cruncher derives the relationship between two variables by finding the\n",
+      "    scenario which has the closest lead gas timeseries in the database.  The follower\n",
+      "    gas timeseries is then simply copied from the closest scenario.\n",
+      "\n",
+      "    Here, 'closest' is defined as the smallest time-averaged root mean squared (L2)\n",
+      "    difference.\n",
+      "\n",
+      "    .. math::\n",
+      "        RMS = \\left ( \\frac{1}{n} \\sum_{t=0}^n (E_l(t) - E_l^{d}(t))^2 \\right )^{1/2}\n",
+      "\n",
+      "    where :math:`n` is the total number of timesteps in the lead gas' timeseries,\n",
+      "    :math:`E_l(t)` is the lead gas emissions timeseries and :math:`E_l^d(t)` is a lead\n",
+      "    gas emissions timeseries in the database.\n",
+      "    \n"
+     ]
+    }
+   ],
+   "source": [
+    "print(silicone.database_crunchers.DatabaseCruncherRMSClosest.__doc__)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Infilling\n",
+    "\n",
+    "Firstly, let's cut the database down to a size that is comprehensible."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "sr15_data_closest_rms = sr15_data.filter(model=[\"WITCH-GLOBIOM 4.2\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we initialise the type of cruncher we want with the infiller database. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cruncher = silicone.database_crunchers.DatabaseCruncherRMSClosest(sr15_data_closest_rms)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can derive the relationship between e.g. `Emissions|CO2` and `Emissions|VOC` in the infiller data."
```

Comment:
point out clearly which of the two is the leader and which the follower

Reply:
ok, more added.